libgadget/utils/endrun.c

```diff
@@ -214,6 +214,7 @@ endrun(int where, const char * fmt, ...)
     va_start(va, fmt);
     messagev(1, where, fmt, va);
     va_end(va);
+    exit(1);
```

Comment:
This is for?

Reply:
My compiler would warn that some functions, specifically the switch statements in the new cooling code, left variables uninitialised in the default case, because it didn't know that endrun never returned. Putting the exit(1) in marks the function as __attribute__(noreturn) and stops the compiler warning.

Reply:
I'll add a comment.